docs/examples.md

```diff
@@ -0,0 +1,57 @@
+# Examples generation
```

Comment:
@philsturgeon Can you start checking this out?

Reply:
Title case for titles please